volatility/framework/plugins/windows/malfind.py

```diff
@@ -88,8 +90,10 @@ def list_injections(
             return
 
         proc_layer = context.layers[proc_layer_name]
+        output_dir = self.config.get('output_dir', None)
```

Comment:
This has changed `list_injections` from being usable by other plugins, to being only usable by this plugin.  This seems like quite a restrictive change (and may impact other plugins, and would require an increase of the plugin's major version number since it's a removal of functionality).  This all seems in aid of getting `self.config` rather than accepting a `dump` field.  Since that's unlikely to be something other plugins care about, I'd sooner it were in the `_generator` method, which gets to see each vad as well, and so can do the dumping and see the config option directly.  #290 should show examples of this...